src/prefect/blocks/core.py

```diff
@@ -72,6 +72,9 @@
 ResourceTuple = Tuple[Dict[str, Any], List[Dict[str, Any]]]
 
 
+_collections_loaded = False
```

Comment:
just as a matter of style consistency, could you make this all upper case?

Reply:
Totally down to use uppercase here. I would normally only use uppercase for a constant, while this is a mutable global variable. Do we normally uppercase those? If so, I can update my mental model.

Reply:
Yea I see what you're saying -- we do use the uppercase convention elsewhere though for mutable values, e.g.:
- https://github.com/PrefectHQ/prefect/blob/main/src/prefect/logging/configuration.py#L24
- https://github.com/PrefectHQ/prefect/blob/main/src/prefect/settings.py#L1859

Reply:
Makes sense to me, and I'm down to keep it consistent.